rules/ref/path/dev-shm.yara

```diff
@@ -39,7 +39,17 @@ rule dev_shm_hidden : critical {
     hash_2023_OrBit_f161 = "f1612924814ac73339f777b48b0de28b716d606e142d4d3f4308ec648e3f56c8"
     hash_2023_Unix_Downloader_Rocke_228e = "228ec858509a928b21e88d582cb5cfaabc03f72d30f2179ef6fb232b6abdce97"
   strings:
-    $dev_shm = /\/dev\/shm\/\.[%\w\.\-\/]{0,64}/
+    $dev_shm = /\/dev\/shm\/\.[\%\w\.\-\/]{0,64}/
+    $ignore_mkstemp = /\/dev\/shm\/\.[%\w\.\-\/]{0,64}X{6}/
+  condition:
+    $dev_shm and not $ignore_mkstemp
+}
+
+rule dev_shm_hidden_mkstemp : high {
```

Comment:
Let's call this a medium for now, at least until we see some malware abuse.

Also, let's change the rule to match all mkstemp style usage, not just hidden ones. We can split them up later if we need to elevate risk for one or the other.